src/typings/BUILD.bazel

```diff
@@ -0,0 +1,8 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@build_bazel_rules_typescript//:defs.bzl", "ts_library")
+
+ts_library(
```

Comment:
I'm not sure this merits a whole separate bazel package - why not put the shim in something like `typings.d.ts` in the `src/` directory?

Reply:
That would work. I like having them in their own folder but a single `typings.d.ts` file will do the trick.